index.html

```diff
@@ -1,133 +1,167 @@
-              <div class="partners">
-                <img src="Images/pngwing.com.png" alt="airbnb" class="logo" id="airbnb">
-              </div>
-            </div>
-          </section>
-    </main>
-</body>
-<footer class="hide">
-    <div id="logo-footer">
-      <img src="Images/torii (1).png" alt="logo" class="logo">
-      <h1>Japan</h1>
-    </div>
-    <div id="footer-info">
-      <p>2023 National Gobernment of Japan.</p>
-      <p>Some Rights Reserved.</p>
-    </div>
-  </footer>
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" href="styles.css" />
+		<!-- Web icon -->
+		<link rel="icon" href="Images/torii (1).png" />
+		<script src="https://kit.fontawesome.com/5211805414.js" crossorigin="anonymous"></script>
+		<!-- Font Awesome -->
+		<script src="Javascript/Activities.js" defer></script>
+		<!-- Activities section -->
+		<script src="Javascript/menu.js" defer></script>
+		<!-- Mobile Menu -->
+		<title>Japanese Culture Festival</title>
+	</head>
+	<body>
+		<header>
+			<div id="mobile-menu">
+				<div id="x-container">
+					<div class="x"></div>
+					<div class="x"></div>
+					<div class="x"></div>
+				</div>
+				<ul id="menu">
+					<li class="menu-sections"><a href="index.html">Home</a></li>
+					<li class="menu-sections"><a href="About.html">About</a></li>
+					<li class="menu-sections">Program</li>
+					<li class="menu-sections">Join</li>
+					<li class="menu-sections">Sponsor</li>
+					<li class="menu-sections">News</li>
+					<li class="menu-sections">Campaign</li>
+				</ul>
+			</div>
+			<div id="hamburger-container">
+				<div class="hamburguer"></div>
+				<div class="hamburguer"></div>
+				<div class="hamburguer"></div>
+			</div>
+			<div id="hiden-header">
+				<div id="social-media">
+					<ul id="social-list">
+						<li class="social"><i class="fa-brands fa-facebook-f"></i></li>
+						<li class="social"><i class="fa-brands fa-twitter"></i></li>
+						<li class="social">English</li>
+						<li class="social">My Page</li>
+						<li class="social">Log Out</li>
+					</ul>
+				</div>
+				<nav id="nav-bar">
+					<a href="index.html">
+						<div id="logo-container">
+							<img src="Images/torii (1).png" alt="logo" class="logo" />
+							<div id="logo-subContainer">
+								<h1>Japanese</h1>
+								<h3>Culture Festival 2023</h3>
+							</div>
+						</div>
+					</a>
+					<ul id="nav-list">
+						<li class="nav-sections"><a href="About.html">About</a></li>
+						<li class="nav-sections">Program</li>
+						<li class="nav-sections">Join</li>
+						<li class="nav-sections">Sponsor</li>
+						<li class="nav-sections">News</li>
+						<li class="nav-sections">Campaign</li>
+					</ul>
+				</nav>
+			</div>
+		</header>
+		<main>
+			<section id="title">
+				<div>
+					<h2>こんにちは世界</h2>
+					<h1>Japanese Culture Festival 2023</h1>
```

Comment:
Please a good HTML semantics rule is that `h1` tag should come before the `h2` tag in all your pages so as to help in Search engine optimization 👍